File: app/src/main/java/com/teslyuk/androidtutorial_sqlitedatabase/db/LogDataInterface.java
package com.teslyuk.androidtutorial_sqlitedatabase.db;

import com.teslyuk.androidtutorial_sqlitedatabase.model.LogModel;

import java.util.List;

public interface LogDataInterface {
    List<LogModel> getAll();

    void addLog(LogModel model);

    void removeLog(LogModel model);

    void open();

    void close();
}

File: app/src/main/java/com/teslyuk/androidtutorial_sqlitedatabase/db/LogDataInterface.kt
package com.teslyuk.androidtutorial_sqlitedatabase.db

import com.teslyuk.androidtutorial_sqlitedatabase.model.LogModel

interface LogDataInterface {
    fun getAll(): List<LogModel>

    fun addLog(model: LogModel)

    fun removeLog(model: LogModel)

    fun open()

    fun close()
}
